v-next/hardhat/test/internal/builtin-plugins/network-manager/json-rpc-request-modifiers/helpers.ts

```diff
@@ -0,0 +1,24 @@
+import type { JsonRpcRequest } from "../../../../../src/types/providers.js";
+
+import { assertHardhatInvariant } from "@ignored/hardhat-vnext-errors";
+
+export function createJsonRpcRequest(
+  method: string,
+  params?: unknown[] | object,
+): JsonRpcRequest {
+  return {
+    jsonrpc: "2.0",
+    id: 1,
+    method,
+    params: params ?? [],
+  };
+}
+
+export function getParams(jsonRpcRequest: JsonRpcRequest): any[] | unknown[] {
```

Comment:
can't we use the `getParams` defined in `v-next/hardhat/src/internal/builtin-plugins/network-manager/json-rpc-request-modifiers/utils.ts`?

Reply:
Done in [this commit](https://github.com/NomicFoundation/hardhat/pull/5810/commits/c16322151cbb6aedbc5b8ff592dd5e36a505eaca)